### DISCUSSION TITLE
how can i use markmap-cli to convert .md to .html file on windows?

### DISCUSSION THREAD

**Discussion by lizhihome** [markmap/markmap#225] [Q&A] [answered]
i have markmap installed on my windows vscode, i like to use markmap-cli on command, how to do it on windows OS?
thanks

**Comment by gera2ld** [comment]
To use `markmap-cli` you need to install it. It has nothing to do with the vscode extension.

For example, to install it with npm:

```bash
npm i -g markmap-cli
```

Then you can use the command `markmap`.

**Reply by lizhihome** [reply]
@gera2ld Thanks, it's very nice tool.
do I need to install node.js on windows? Now my windows has Apache installed, I don't know if they are conflict.
Thanks,

**Reply by gera2ld** [reply, marked as answer]
Yes, it is written in node.js so you need to install it.
Node.js has nothing to do with apache.
